src/utils/inputValidation/__tests__/attribute.spec.ts

```diff
@@ -0,0 +1,18 @@
+import { validateAttribute } from '../attribute';
+import { loggerMock } from '../../../logger/__tests__/sdkLogger.mock';
+
+
+test('INPUT VALIDATION for Attribute', () => {
+  expect(validateAttribute(loggerMock, '', 'empty', 'some_method_attrs')).toEqual(false); // It should be invalid if the attribute key is not a string
```

Comment:
Emma, it seems that there are more asserts [here](https://github.com/splitio/javascript-client/blob/development/src/utils/__tests__/inputValidation/attribute.spec.js) that we can also migrate.